### DISCUSSION TITLE
Extreme Nausea does not show up in red font

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#17597] [Feature requests]
### Operating System

Windows 11 Pro

### OpenRCT2 build

OpenRCT2, v0.4.1-32-g5688eb5 (5688eb5 on develop) provided by GitHub

### Describe the issue

When testing a ride's stats, an Extreme Nausea rating is not Displayed in red font

### Area(s) with issue?

The bug also exists in RCT2 (vanilla)

### Steps to reproduce

1. Build Extreme Nausea Ride
2. Test ride
3. View stats in stats tab of ride window

### Attachments

![Invasion Of The Peep Snatchers 2022-07-22 22-10-19](https://user-images.githubusercontent.com/94884086/180587988-f9c45eb4-8f89-4638-8efc-10693bcb1351.png)

**Comment by duncanspumpkin** [comment]
This isn't really a bug though its just how the game is designed to draw attention to some key items.
